app/search_builders/curation_concerns/file_set_search_builder.rb

```diff
@@ -2,9 +2,9 @@ module CurationConcerns
   class FileSetSearchBuilder < ::SearchBuilder
     include CurationConcerns::SingleResult
 
-    # This overrides the filter_models in FilterByType
-    def filter_models(solr_parameters)
-      solr_parameters[:fq] << ActiveFedora::SolrQueryBuilder.construct_query_for_rel(has_model: ::FileSet.to_class_uri)
+    # This overrides the models in FilterByType
+    def models
```

Comment:
I'd prefer a method name that made it clear the return is URIs and not model Classes or some other representation of models.  This is a complaint for FilterByType, I guess, but that method doesn't exist in my branch, so it must only have been added recently.